### PR TITLE
Add Newspack WordPress plugin

### PR DESCRIPTION
**Pull request by adamsilverstein**
Project URL: https://github.com/Automattic/newspack-plugin 
Description: The Newspack plugin provides Newspack: an open-source publishing platform built on WordPress for small to medium sized news organizations. It is an “opinionated” platform that stakes out clear, best-practice positions on technology, design, and business practice for news publishers. 
Icon: NewspackLogo.png

Example URLs:
1. https://cowgirlmagazine.com/	
2. https://www.hechingerreport.org/	
3. https://www.oaklandside.org/	
4. https://www.calmatters.org/	
5. https://www.blackvoicenews.com/

Check for:
- a href: `newspack.pub class=imprint` - html check works in AMP context as well.

### CODE REVIEW

File: src/technologies.json [hunk 1]
```diff
@@ -14376,6 +14376,18 @@
       "saas": true,
       "website": "https://newrelic.com"
     },
+    "Newspack": {
+      "cats": [
+        87
+      ],
+      "description": "Newspack is an open-source publishing platform built on WordPress for small to medium sized news organizations. It is an “opinionated” platform that stakes out clear, best-practice positions on technology, design, and business practice for news publishers.",
+      "dom": "link[href*='newspack.pub']",
```
Comment (westonruter) [inline src/technologies.json:14384]: Is this missing the `class`?

```suggestion
      "dom": "link.imprint[href*='newspack.pub']",
```

Comment (AliasIO) [reply src/technologies.json:14384]: I think this should be `"dom": "a.imprint[href*='newspack.pub']"`.

File: src/technologies.json [hunk 2]
```diff
@@ -14376,6 +14376,18 @@
...
+      "html": [
+        "<a href=\"https://newspack.pub/\" class=\"imprint\">"
+      ],
```
Comment (AliasIO) [inline src/technologies.json:14387]: `html` has been deprecated in favour of `dom`, please remove this line.

**Comment by AliasIO** [comment]
Thank you! Could you resize the icon to 32x32px or provide an SVG (preferred)?

File: src/technologies.json [hunk 3]
```diff
@@ -14376,6 +14376,18 @@
       "saas": true,
       "website": "https://newrelic.com"
     },
+    "Newspack": {
```
[the remark this answers is not in the document]
Comment (AliasIO) [reply src/technologies.json:14379]: Please also add `"oss": true` as the software is open-source.

[the remark this answers is not in the document]
Comment (AliasIO) [reply src/technologies.json:14379]: Probably also add `"implies": "Newspack by Automattic"`.

**Comment by AliasIO** [comment]
Thanks, if WordPress is not strictly required then it doesn't need to be added. :)

**Pull request merged**